fbgemm_gpu/src/quantize_ops.cu

```diff
@@ -109,7 +109,9 @@ __global__ inline void _float_to_fused8bitrowwise_cuda_kernel(
 template <typename T>
 __device__ inline __attribute__((always_inline)) T
 quantize_ops_shfl_xor(const T val, int laneMask, int width) {
-#if CUDA_VERSION >= 9000
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
Can we merge this condition to avoid duplicated `__shfl_xor(val, laneMask, width);`?

Reply:
The conditions were combined and aligned with the ones in `fbgemm_cuda_utils.cuh`.